src/test/kotlin/graphics/scenery/tests/examples/volumes/VolumeSamplingExample.kt

```diff
@@ -58,8 +57,10 @@ class VolumeSamplingExample: SceneryBase("Volume Sampling example", 1280, 720) {
             if(file != null) {
                 files.add(file.absolutePath)
                 VolumeType.File
-            } else {
-                VolumeType.Procedural
+            }
+            else {
+                println("procedural now")
```

Comment:
`println` should not be used, use `logger.info()` instead if necessary, also code formatting :-)